website/addons/owncloud/tests/test_models.py

```diff
@@ -0,0 +1,88 @@
+from nose.tools import *  # noqa
+import mock
+
+from tests.base import get_default_metaschema
+from framework.auth.decorators import Auth
+
+from website.addons.base.testing import models
+
+from website.addons.owncloud.model import AddonOwnCloudNodeSettings
+from website.addons.owncloud.tests.factories import (
+    OwnCloudAccountFactory, OwnCloudNodeSettingsFactory,
+    OwnCloudUserSettingsFactory
+)
+from website.addons.owncloud.tests import utils
+
+class TestNodeSettings(models.OAuthAddonNodeSettingsTestSuiteMixin, utils.OwnCloudAddonTestCase):
+
+    short_name='owncloud'
+    ExternalAccountFactory = OwnCloudAccountFactory
+    NodeSettingsFactory = OwnCloudNodeSettingsFactory
+    NodeSettingsClass = AddonOwnCloudNodeSettings
+    UserSettingsFactory = OwnCloudUserSettingsFactory
+
+
+    def _node_settings_class_kwargs(self, node, user_settings):
+        return {
+            'user_settings': self.user_settings,
+            'folder': '/Documents',
+            'owner': self.node,
+            'node': self.node
+        }
+
+    def setUp(self):
```

Comment:
I think this override might be superfluous -- the base class should already be calling `set_node_settings`